### PR TITLE
Canvas image module 

### PR DESCRIPTION
**Pull request by DanielMuckelbauer**
Not sure about that http call in the example, since it requires an internet connection. 
I think that was prefereable to adding some asset just to have the image for the example, but tell me what you think.

### CODE REVIEW

File: src/spectrecoff-cli/commands/CanvasImage.fs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        Many [
+            CO [
+                C "This submodule provides functionality from the canvas image widget of Spectre.Console ("
+                Link "https://spectreconsole.net/widgets/canvas-image"
+                C ")"
+            ]
+            NL
+            C "The canvas image can be used using the canvasImage function:"
+            BI [
+                P "canvasImage: ImageSource -> CanvasImage"
```
Comment (EluciusFTW) [inline src/spectrecoff-cli/commands/CanvasImage.fs:54]: The documentation of the extension method to transform it into a payload is missing.

File: src/spectrecoff/CanvasImage.fs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+open System
+open System.IO
+open Spectre.Console
+
+let mutable maxWidth = 16
+
+type ImageSource =
+| Bytes of Byte[]
+| Stream of Stream
+| Path of String
```
Comment (EluciusFTW) [inline src/spectrecoff/CanvasImage.fs:13]: Not Uri, or does Spectre also just accept a string?

**Review by EluciusFTW — COMMENTED** [review]
Concerning the http call to get an example, why not just take the progress png from the sample folder instead?

**Review by EluciusFTW — APPROVED** [review]

File: src/spectrecoff/CanvasImage.fs (hunk 2 above)
Comment (DanielMuckelbauer) [reply src/spectrecoff/CanvasImage.fs]: As discussed, no URI.

File: src/spectrecoff-cli/commands/CanvasImage.fs (hunk 1 above)
Comment (DanielMuckelbauer) [reply src/spectrecoff-cli/commands/CanvasImage.fs:56]: Done

**Pull request merged**